api/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -1433,4 +1439,22 @@ public String toString() {
     }
   }
 
+  /** A picker that always returns the same result. */
+  public static final class FixedResultPicker extends SubchannelPicker {
+    private final PickResult result;
+
+    public FixedResultPicker(PickResult result) {
```

Comment:
Want to add that to the API review meeting? I think we had liked the idea of not reproducing the PickResult API into the FixedResultPicker.

Most policies only use the FixedResultPicker once. CdsLoadBalancer2 and ClusterResolverLoadBalancers each use it three times. The explicitness seems fair, also since this should end up being used for "withNoResult".